server_api/external.py

```diff
@@ -60,8 +60,13 @@ def CheckFollower(server, author, follower):
     elif server.server_address == "https://c404-social-distribution.herokuapp.com/service/":
         follower = requests.get(f"{server.server_address}authors/{author}/followers/{follower}", auth=HTTPBasicAuth(server.auth.split(":")[0], server.auth.split(":")[1]))
         follower_json = follower.json()
-    # print(follower_json)
+    
     return follower_json
+
+def delete_follower(server, author, follower):
```

Comment:
maybe change this to author_id and follower_id for better clarity

Reply:
changed, you can check it again